.circleci/images/primary/README.md

```diff
@@ -2,20 +2,17 @@
 
 This folder contains Dockerfiles for all versions of Ruby used in the testing and development of dd-trace-rb.
 
-## Multiplatform builds
+## Multi-arch images
 
-Images marked with a "# This image supports multiple platforms" are able to be built for both amd64 (x86_64) and
-arm64 (aarch64) Linux.
+All images are able to be built for both `x86_64` (a.k.a `amd64`) and `aarch64` (a.k.a `arm64/v8`) Linux.
 
-Here's an example of building the Ruby 3.1 image:
+Here's an example of manually building Ruby 3.1 images:
 
 ```bash
-# To build and push the image (update tag as needed):
-$ docker buildx build . --platform linux/amd64,linux/arm64/v8 -f Dockerfile-3.1.1 -t ivoanjo/docker-library:ddtrace_rb_3_1_1 --push
+# To build single-arch images locally (NEVER PUSH THESE!):
+$ docker buildx build . --platform linux/x86_64 -f Dockerfile-3.1.1 -t ghcr.io/datadog/dd-trace-rb/ruby:3.1.1-dd
+$ docker buildx build . --platform linux/aarch64 -f Dockerfile-3.1.1 -t ghcr.io/datadog/dd-trace-rb/ruby:3.1.1-dd
 
-# The tag created will automatically contain both architectures. To additionally create an
-# architecture-specific tag, I needed to do the following (sha is the specific image)
-$ docker pull ivoanjo/docker-library:ddtrace_rb_3_1_1@sha256:56402a1c5e522b669965db4600f1a4fa035f6e3597d098ec808e77192c4238fd
-$ docker tag ivoanjo/docker-library@sha256:56402a1c5e522b669965db4600f1a4fa035f6e3597d098ec808e77192c4238fd ivoanjo/docker-library:ddtrace_rb_3_1_1_amd64
-$ docker push ivoanjo/docker-library:ddtrace_rb_3_1_1_amd64
+# To build AND push multi-arch images (but DON'T DO THAT IN GENERAL, unless e.g CI is down):
+$ docker buildx build . --platform linux/x86_64,linux/aarch64 -f Dockerfile-3.1.1 -t ghcr.io/datadog/dd-trace-rb/ruby:3.1.1-dd --push
```

Comment:
Minor: Since people tend to copy-paste commands and not always edit them, I suggest removing the `--push` from the actual command example, and instead just stating on the text "If you ever need to push (....) add --push to the above command".